tic-tac-toe/src/app/components/square/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Square = props => {
```

Comment:
When we define a new component we preferer use function instead of arrow function.
You can write ({ onClick, value }) instead of props, so then you can write directly the params without props